templates/cli/lib/config.js.twig

```diff
@@ -125,27 +125,39 @@ class Config {
 }
 
 class Local extends Config {
-    static CONFIG_FILE_PATH = "{{ spec.title|caseLower }}.json";
+    static CONFIG_FILE_PATH = "{{ spec.title|caseLower }}.config.json";
+    static CONFIG_FILE_PATH_LEGACY = "{{ spec.title|caseLower }}.json";
```

Comment:
As per torsten suggestion, it might be better to use library for this: https://github.com/unjs/c12

that will aslo support .js out of the box, which would be great addition for those wanting a bit of complexity.

I would never try to load such JS/TS configs on my own, trusting lib with 10M monthly downloads is better